FasTnT.Formatter.Xml/Formatters/XmlResponseFormatter.cs

```diff
@@ -26,23 +26,19 @@ public static XElement FormatPoll(PollResponse response)
         return queryResults;
     }
 
-    public static XElement FormatUnexpectedError()
+    public static XElement FormatFault(EpcisException exception)
     {
```

Comment:
After reflection, XML formatter should not contain SOAP-related code. This should remain where the SOAP requests are handled.